code/modules/mob/living/silicon/pai/emote.dm

```diff
@@ -0,0 +1,163 @@
+/mob/living/silicon/pai/emote(var/act,var/m_type=1,var/message = null, var/auto)
+	if(timestopped)
+		return //under effects of time magick
+	var/param = null
+	if (findtext(act, "-", 1, null))
```

Comment:
@Shadowmech88 If you start learning how to use Regex now you'll be six feet under sooner than adequately proficient at Regex.

And if I'm correct, Regex is also usually a bit slower than other methods.